e2e/testdata/fn-render/resource-has-pkgname-prefix/.expected/diff.patch

```diff
@@ -0,0 +1,199 @@
+diff --git a/mysql/mysql-deployment.yaml b/mysql/mysql-deployment.yaml
+index cbbe597..6b307e8 100644
+--- a/mysql/mysql-deployment.yaml
++++ b/mysql/mysql-deployment.yaml
+@@ -11,33 +11,36 @@
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-
+ apiVersion: v1
+ kind: Service
+ metadata:
+-  name: wordpress-mysql
++  annotations:
++    abc: def
++    foo: bar
++    projectId: PROJECT_ID # kpt-set: ${gcloud.core.project}
++    teamname: YOURTEAM # kpt-set: ${teamname}
+   labels:
+     app: wordpress
+-  annotations:
```

Comment:
there's some unrelated formatting here such as reordering of the fields which obfuscates what the fuction is doing.

Reply:
Seems like it is related to the functions and not `render` itself. `set-annotation`, `set-namespace`, `set-label` functions format the resources.

@Shell32-Natsu